thunder/images/images.py

```diff
@@ -79,35 +80,44 @@ def toblocks(self, size='150', padding=None):
         from thunder.blocks.blocks import Blocks
         from thunder.blocks.local import LocalChunks
 
+        if chunk_size is 'auto':
+            chunk_size = str(int(1000000.0/self.shape[0]))
```

Comment:
Yeah, let's make sure we get this right :)

I'm using the following terminology:
a "chunk" is a subset of (x, y, z)
a "block" is (t, x, y, z), where you have an (x, y, z) chunk and all time points

The block is the intermediate object in `Images.toseries` and `Images.map_as_series`. Bolt works with chunks, so if we want the blocks to be of size S=100MB when you choose `auto`, then the chunk that we should use is 100MB/len(t). That's the computation that's being done in line `84`.

Does that make sense?